src/cmd/flux-jobs.py

```diff
@@ -95,39 +76,7 @@ def validate(self, path, config):
 
 
 def load_format_string(args):
-    if "{" in args.format:
-        return args.format
-
-    config = FluxJobsConfig().load()
-
-    if args.format == "help":
-        print("\nConfigured flux-jobs output formats:\n")
-        for name, entry in config.formats.items():
-            print(f"  {name:<12}", end="")
-            try:
-                print(f" {entry['description']}")
-            except KeyError:
-                print()
-        sys.exit(0)
-    elif args.format.startswith("get-config="):
-        _, name = args.format.split("=", 1)
-        try:
-            entry = config.formats[name]
-        except KeyError:
-            LOGGER.error("--format: No such format %s", name)
-            sys.exit(1)
-        print(f"[formats.{name}]")
-        try:
-            print(f"description = \"{entry['description']}\"")
-        except KeyError:
-            pass
-        print(f"format = \"{entry['format']}\"")
-        sys.exit(0)
-    try:
-        return config.formats[args.format]["format"]
-    except KeyError:
-        LOGGER.error("--format: No such format %s", args.format)
-        sys.exit(1)
+    return FluxJobsConfig().load().get_format_string(args.format)
```

Comment:
why not just remove `load_format_string()` and have callers call `FluxJobsConfig().load().get_format_string(args.format)` directly?

(if reason is in later commit, ignore).

Reply:
ok, AFAICT there's just the one caller, so didn't seem necessary to keep the function.

Reply:
Thanks, I'll take a look at it.

Reply:
Ok, looking at it you're right. I'll remove.

Reply:
fixed and force-pushed